test/SpokePoolClient.ValidateFill.ts

```diff
@@ -469,9 +475,13 @@ describe("SpokePoolClient: Fill Validation", function () {
 
     // Override the first spoke pool deposit ID that the client thinks is available in the contract.
     await spokePoolClient1.update();
-    spokePoolClient1.firstDepositIdForSpokePool = 1;
+    spokePoolClient1.firstDepositIdForSpokePool = deposit.depositId + 1;
     expect(fill.depositId < spokePoolClient1.firstDepositIdForSpokePool).is.true;
-    await queryHistoricalDepositForFill(spokePoolClient1, fill);
+    const search = await queryHistoricalDepositForFill(spokePoolClient1, fill);
+    expect(search.found).is.false;
+    if (search.found === false) {
+      expect(search.code).to.equal(InvalidFill.DepositIdInvalid);
```

Comment:
Reminder to also test for `DepositIdNotFound` and `FillMismatch`.

Reply:
Tests were updated to verify `FIllMismatch`. `DepositIdNotFound` is currently difficult to test but I am working on that downstream in `relayer-v2`, and those tests might be backported to the SDK.